Update upgrading-always-on-availability-group-replica-instances.md (#9164)
MicrosoftDocs/sql-docs: docs/database-engine/availability-groups/windows/upgrading-always-on-availability-group-replica-instances.md

>
> If `REQUIRED_SYNCHRONIZED_SECONDARIES_TO_COMMIT` is set to either `1` or `2`, the Primary replica may be unavailable for read/writes when a corresponding number of sync secondary replicas are not available during the update process.

> [!NOTE]  

> > In case of Upgrade in-place on a secondary replica, which upgrades the instance to a newer version, the databases inside the AG will remain on state Synchronized / In Recovery. After a planned manual failover on the AG, the recovery will finish and the databases will be upgraded.

Reviewer: ```suggestion
> When you perform an in-place upgrade of a secondary replica to a newer version of SQL Server, the database inside the availability group remains in the **Synchronized / In Recovery** state until the availability group is manually failed over, which finishes the recovery and upgrades the database.   
```

Reviewer: @asavioliMSFT please commit the suggestion if you agree my changes haven't technically changed the meaning of your addition. thanks!

Author: Thanks @MashaMSFT. I agree with your suggestion to change the text

> > When you perform an in-place upgrade of a secondary replica to a newer version of SQL Server, the database inside the availability group remains in the **Synchronized / In Recovery** state until the availability group is manually failed over, which finishes the recovery and upgrades the database.   

Reviewer: ```suggestion
> When you perform an in-place upgrade of a secondary replica to a newer version of SQL Server, the database inside the availability group remains in the **Synchronizing / In recovery** or **Synchronized / In Recovery** state until the availability group is manually failed over, which finishes the recovery and upgrades the database.  An upgraded primary replica can no longer ship logs to any lower version secondary replica and data movement stops and no automatic failover can occur for that replica, and your availability databases are vulnerable to data loss. After you upgrade the old primary, you may need to resume synchronization.
```

> > When you perform an in-place upgrade of a secondary replica to a newer version of SQL Server, the database inside the availability group remains in the **Synchronizing / In recovery** or **Synchronized / In Recovery** state until the availability group is manually failed over, which finishes the recovery and upgrades the database.  An upgraded primary replica can no longer ship logs to any lower version secondary replica and data movement stops and no automatic failover can occur for that replica, and your availability databases are vulnerable to data loss. After you upgrade the old primary, you may need to resume synchronization.

Reviewer: ```suggestion
> When you perform an in-place upgrade of a secondary replica to a newer version of SQL Server, the database inside the availability group remains in the **Synchronizing / In recovery** or **Synchronized / In Recovery** state until the availability group is manually failed over, which finishes the recovery and upgrades the database.  An upgraded primary replica can no longer ship logs to any lower version secondary replica and data movement stops and no automatic failover can occur for that replica, and your availability databases are vulnerable to data loss. After you upgrade the old primary, you may need to resume synchronization. It is recommended to upgrade all secondary replicas before failing over to a replica with the new version.  That way you have the option of doing a failover after the database(es) are upgraded to the new format.
```
